scripts/docker-entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+# build sphinx config with current environment
+cat conf/*.part > conf/sphinx.conf.in
+envsubst < conf/sphinx.conf.in > conf/sphinx.conf
+
+# copy sphinx config and wordforms from github / image content into docker volume
+cp -f conf/sphinx.conf /etc/sphinxsearch/sphinx.conf
+cp -f conf/wordforms_main.txt /etc/sphinxsearch/wordforms_main.txt
```

Comment:
Could this not be done in dockerfile during the build instead of during each docker start ?

Reply:
the reason for having this here is that we can use the same image for dev, int and prod staging. envsubst will replace the db suffix etc. when run the image/container. otherwise we would have to build one image per staging.

Reply:
OK sorry I missed the envsubst part above. That make now total sense.